docs/_articles/en/einstein/einstein-troubleshoot.md

```diff
@@ -5,4 +5,33 @@ lang: en
 
 Generative AI uses large language models (LLMs) to generate the output. Because LLMs are trained by other organizations, the output isn’t always what you expect, and the code that’s generated isn’t perfect. If the output doesn’t meet your expectations, you can generate a new output by starting over. The generated output from the previous attempt isn’t saved, and the new output replaces it.
 
-To keep track of all known issues and potential workarounds, use the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) tab in this repo.
+To track all known issues and potential workarounds, use the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) tab in this repo.
+
+**Note:** If your company requires the use of a proxy, we recommend setting this through the Salesforce CLI, not through VS Code, in order to use the Einstein for Developers extension.
+
+## Troubleshooting Checklist
+Use this troubleshooting checklist to identify and eliminate common problems with using Einstein for Developers.
+
+### Installation Checklist
+
+1. Confirm that you have the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) or the [Salesforce Expanded Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-expanded) installed.
+2. Check that you are on VS Code version 1.82.0 or higher. 
+3. Check that your Apex Extension is version 59.8.0 or higher.
+4. Hover over {} in the status bar to view the status of the Apex Language Server and make sure it’s running.  
+5. Einstein for Developers needs Salesforce CLI version 59.13.0 or higher. Make sure your CLI version hasn’t fallen out of sync. See [Salesforce CLI](https://developer.salesforce.com/tools/salesforcecli). 
+
+### Development Environment Checklist
+
+1. Einstein for Developers only runs inside a Salesforce project. Confirm that you have a Salesforce DX project open. 
+2. Check that your Einstein for Developers extension is activated using the **Developer: Show Running Extensions** command. If your extension isn’t activating, file an [issue](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) in our GitHub repo.
+3. Make sure you’re connected to a Salesforce org. The status of your default org connection is visible in the status bar.
+
+
+![Default Org](./images/default-org.png)
+
+
+**Tip**: Run the **SFDX: Deploy This Source to Org** command to easily check that your Salesforce extensions are running correctly and that you're connected to a Salesforce org.
```

Comment:
I'm not so sure about this tip.  Could a first-time user potentially deploy some  edited local code over a remote file without realizing what they were doing (if this is the first thing they are trying after installation)?  Do we have any other commands that simply read something from the org and report it?  Something with less potential to do a destructive change accidentally?  Or at least maybe we include some caveat here: "if you are connected to a scratch org and don't have any local mods to an Apex class, try running SFDX: Deploy..."

Reply:
Agreed. I think deploying is a little heavy handed, but we couldn't think of a different command. I could remove the tip...

Reply:
Great feedback Ken! I opened up VS Code to find something less dangerous - how about `SFDX: Open Default Org`? That both educates the user on a command they might not know about and doesn't put them in a risky position for having run it.